src/pytds/tls.py

```diff
@@ -105,11 +105,19 @@ def validate_host(cert, name):
         ext = cert.get_extension(i)
         if ext.get_short_name() == b'subjectAltName':
             s = str(ext)
-            # SANs are usually have form like: DNS:hostname
-            if s.startswith('DNS:') and s[4:] == s_name:
-                return True
-
-    # TODO handle wildcards
+            items = s.split(',')
+            for item in items:
+                dnsentry = item.strip()
+                # SANs are usually have form like: DNS:hostname
+                if dnsentry.startswith('DNS:') and s[4:] == s_name:
+                    return True
+                if dnsentry.startswith('DNS:*.'):  # support for wildcards, but only at the first position
```

Comment:
It would be better to extract SAN host name suffix before previous condition, and then check host name for exact match and for wildcard match.  Instead of checking for `DNS:` prefix twice

Reply:
I added this stuff but I'm not sure I got the tests right. How do I execute tests locally?